.github/workflows/run_data_sync.yml

```diff
@@ -105,11 +105,11 @@ jobs:
           python run_page/coros_sync.py ${{ secrets.COROS_ACCOUNT }} ${{ secrets.COROS_PASSWORD }}
 
       - name: Run sync Keep_to_strava script
-        if: env.RUN_TYPE == 'keep_to_strava'
+        if: env.RUN_TYPE == 'keep_to_strava_sync'
         run: |
           python run_page/keep_to_strava_sync.py ${{ secrets.KEEP_MOBILE }} ${{ secrets.KEEP_PASSWORD }} ${{ secrets.STRAVA_CLIENT_ID }} ${{ secrets.STRAVA_CLIENT_SECRET }} ${{ secrets.STRAVA_CLIENT_REFRESH_TOKEN }} --sync-types running cycling hiking
         # If you only want to sync `type running` modify args --sync-types running, default script is to sync all data (rides, hikes and runs).
-        # If you only want to sync data from keep to strava, and the running page display the data from strava, you need to change the "if: env.RUN_TYPE == 'keep_to_strava'" -> "if: env.RUN_TYPE == 'strava'"
+        # If you only want to sync data from keep to strava, and the running page display the data from strava, you need to change the "if: env.RUN_TYPE == 'keep_to_strava_sync'" -> "if: env.RUN_TYPE == 'strava'"
```

Comment:
Is the step of changing to 'strava' outdated and unnecessary in the latest version of the code? Please update the comments